pkg/controllers/disruption/drift_test.go

```diff
@@ -78,6 +78,47 @@ var _ = Describe("Drift", func() {
 		})
 		nodeClaim.StatusConditions().MarkTrue(v1beta1.Drifted)
 	})
+	Context("Metrics", func() {
+		const eligibleNodesMetric = "karpenter_disruption_eligible_nodes"
+		var eligibleNodesLabels = map[string]string{
+			"method":             "drift",
+			"consolidation_type": "",
```

Comment:
IIUC not specifying it would result in us returning the first metric that has the method set to `"drift"`. That's fine in this case since there is only one but I wanted to be explicit. This also tests that we shouldn't set the consolidation_type for this metric.